scripts/plotting/polar_map.py

```diff
@@ -245,6 +246,14 @@ def polar_map(adfobj):
                             # difference: each entry should be (lat, lon)
                             dseasons[s] = mseasons[s] - oseasons[s]
                             dseasons[s].attrs['units'] = mseasons[s].attrs['units']
+                            
+                            # percent change
+                            #pseasons[s] = (abs(mseasons[s] - oseasons[s])/((mseasons[s] + oseasons[s])/2.0))*100.0 #absolute 
```

Comment:
Could you remove this commented out line?

Reply:
Done